pkg/controller/mizar/util.go

```diff
@@ -80,6 +84,9 @@ func ConvertToPodContract(pod *v1.Pod) *BuiltinsPodMessage {
 		network = ""
 	}
 
+	klog.Infof("Pod Name: %s, HostIP: %s, Namespace: %s, Tenant: %s, Arktos network: %s",
```

Comment:
Do we need to set a log level for these logs?

Reply:
Good call, I will add log level

Reply:
@clu2xlu Are we still planning to fix this and update the PR?